### PR TITLE
send mindustry to brazil

### PR DESCRIPTION
**Pull request by DeltaNedas**
tinanium press

### CODE REVIEW

**Review by Anuken — APPROVED** [review]

**Review by summetdev — APPROVED** [review]

**Comment by DeltaNedas** [comment]
whats taking travis so long its been 20 mins

**Comment by summetdev** [comment]
imgonline-com-ua-PicOnPic-bOdzwBJkPSd

**Comment by Quezler** [comment]
is this some peasant joke i do not understand?

**Comment by ThePythonGuy3** [comment]
i just dont get it, but whatever. 🇧🇷

**Comment by Pietro303HD** [comment]
tinanium won

**Comment by DeltaNedas** [comment]
holy shit travis

**Comment by markozajc** [comment]
travis took so long because he had a trip to brazil

**Comment by z0mbiesrock** [comment]
bad egg cat

**Comment by Catchears** [comment]
> is this some peasant joke i do not understand?

Kind of

**Review by Catchears — APPROVED** [review]
10/10 would merge

**Review by ThePythonGuy3 — APPROVED** [review]

**Review by Ilya246 — APPROVED** [review]

**Comment by summetdev** [comment]
~~Quezler could you approve these changes~~

**Comment by BlueWolf3682** [comment]
nani

**Comment by DeltaNedas** [comment]
@Quezler please review

**Review by ILiekMelons — APPROVED** [review]
brazil

**Review by QmelZ — APPROVED** [review]

**Comment by QmelZ** [comment]
amazing lmao

**Comment by ThePythonGuy3** [comment]
i love how #github is getting bloated of reviews lmao

**Review by braainsiogame — APPROVED** [review]

**Review by Quezler — APPROVED** [review]

**Comment by summetdev** [comment]
We got it.

**Comment by ThePythonGuy3** [comment]
oh yes

**Review by BlueWolf3682 — COMMENTED** [review]
yes

**Comment by braainsiogame** [comment]
@BlueWolf3682 APPROVE IT

**Comment by Pogreu** [comment]
Ah não

**Review by Pogreu — APPROVED** [review]

**Review by Ilya246 — APPROVED** [review]

**Review by Ilya246 — APPROVED** [review]

**Review by braainsiogame — APPROVED** [review]

**Review by QmelZ — APPROVED** [review]

**Review by braainsiogame — APPROVED** [review]

**Review by braainsiogame — APPROVED** [review]

**Review by braainsiogame — APPROVED** [review]

**Review by QmelZ — APPROVED** [review]
third time is a charm

**Comment by Pogreu** [comment]
Anuke is charming*

**Comment by braainsiogame** [comment]
why

**Comment by ThePythonGuy3** [comment]
~~will this ever get merged~~

**Review by Catchears — APPROVED** [review]
so subtle yet so game-changing

i love it

**Comment by summetdev** [comment]
Stop

**Review by YellOw139 — CHANGES_REQUESTED** [review]
Also see line 557. Otherwise great PR. Greatly improves the game.

**Review by TheSlaylord — APPROVED** [review]

File: core/assets/bundles/bundle.properties [hunk 1]
```diff
@@ -1064,7 +1064,7 @@ block.scatter.name = Scatter
 block.hail.name = Hail
 block.lancer.name = Lancer
 block.conveyor.name = Conveyor
-block.titanium-conveyor.name = Titanium Conveyor
+block.titanium-conveyor.name = Tinanium Conveyor
 block.plastanium-conveyor.name = Plastanium Conveyor
 block.armored-conveyor.name = Armored Conveyor
 block.armored-conveyor.description = Moves items at the same speed as titanium conveyors, but possesses more armor. Does not accept inputs from the sides from anything but other conveyor belts.
```
[the remark this answers is not in the document]
Comment (DeltaNedas) [reply core/assets/bundles/bundle.properties:1070]: ok will fix very sorry

**Review by genNAowl — APPROVED** [review]

**Comment by the-kido** [comment]
I would be lying if I said I knew what was going on

**Review by Pietro303HD — APPROVED** [review]

**Comment by markozajc** [comment]
brazil man sent me to mindustry
image

**Comment by Erisfiregamer1** [comment]
merge this already

**Comment by Erisfiregamer1** [comment]
OH NOO A CONFLICT OH GOD

**Review by Erisfiregamer1 — APPROVED** [review]
yes

**Comment by the-kido** [comment]
After thoughtful consideration, I deem this worthy of a merge.

**Review by the-kido — APPROVED** [review]

**Review by sample-text-here — APPROVED** [review]
ye es i want to go to brazil

**Review by ItsJonyBoi — APPROVED** [review]
Yes

**Review by notrealn — APPROVED** [review]

**Review by Catchears — APPROVED** [review]

**Comment by TheSlaylord** [comment]
conflicting file detected

**Review by ritzip — APPROVED** [review]

**Review by QmelZ — APPROVED** [review]

**Review by TheMrAlbert — APPROVED** [review]

**Comment by TheMrAlbert** [comment]
Ah yes...

**Review by Erisfiregamer1 — CHANGES_REQUESTED** [review]
There seems to be errors.
I think maybe the sprite names are causing this. It needs to be fixed.
Otherwise 10/10 would merge

**Review by Erisfiregamer1 — APPROVED** [review]
Force a push to the latest  commit, and anooke will mergg

**Review by AJCM-git — APPROVED** [review]
Game changing, thanks you so much for this.

**Review by Volas171 — APPROVED** [review]

**Comment by Violet-Scarelli** [comment]
Ah shit, here we go again.

**Comment by Volas171** [comment]
lol the conflict still broke

**Comment by Volas171** [comment]
: facepalm :

**Review by Erisfiregamer1 — APPROVED** [review]
When do we merge this again?

**Comment by Volas171** [comment]
idk , when the conflict gets fixed maybe

**Comment by Anuken** [comment]
the end

**Comment by DeltaNedas** [comment]
ono pull got stale

**Comment by markozajc** [comment]
no anuke you can't just close this nooo

**Comment by Volas171** [comment]
nooooooo

**Comment by catatagun** [comment]
I say anuke die for this

**Pull request closed without merging**